### PR TITLE
Cudos 1537 cosmostation support

### PR DESCRIPTION
**Pull request by stefanpetkov90**
- Added Cosmostation ledger support
- UI changes to "Connect" page and Navbar to display ledger type

### CODE REVIEW

**Review by stanfilip0v — APPROVED** [review]

File: src/components/Layout/UserInfo.tsx [hunk 1]
```diff
@@ -58,17 +61,30 @@ const UserInfo = () => {
     <Box sx={styles.user} onMouseLeave={() => setOpen(false)}>
       <Box onMouseEnter={() => setOpen(true)} sx={styles.userContainer}>
         <Box sx={styles.userInnerContainer}>
+          <Box>
+            <img
+              src={connectedLedger === 'Keplr' ? KeplrLogo : CosmostationLogo}
```
Comment (SpaghettiOverload) [inline src/components/Layout/UserInfo.tsx]: I am thinking that having the two certain options covered and then defaulting is a bit safer, like:

`                src={
                  connectedLedger === 'Keplr' ? KeplrLogo :
                  connectedLedger === 'Cosmostation' ? CosmostationLogo :
                  WalletIcon
                }`

Also comparing to the predefined constants, such as CosmosNetworkConfig.KEPLR_LEDGER and CosmosNetworkConfig.COSMOSTATION_LEDGER

File: src/App.tsx [hunk 2]
```diff
@@ -40,9 +42,23 @@ const App = () => {
...
+        ledger = await connectKeplrLedger()
+        return ledger
+      case CosmosNetworkConfig.COSMOSTATION_LEDGER:
+        ledger = await connectCosmostationLedger()
+        return ledger
+      default:
+        return { address: '', accountName: '' }
+    }
+  }
+
```
Comment (SpaghettiOverload) [inline src/App.tsx]: I think switchLedgerType() can be a helper function and called when needed, as it is currently duplicated in App.tsx and ConnectWallet.tsx

File: src/ledgers/utils.ts [hunk 3]
```diff
@@ -433,28 +435,44 @@ export const unknownMessage = {
...
+      client = await window.getOfflineSigner(import.meta.env.VITE_APP_CHAIN_ID)
+      return client
+    case CosmosNetworkConfig.COSMOSTATION_LEDGER:
+      client = await cosmostationSigner(import.meta.env.VITE_APP_CHAIN_ID)
+      return client
+    default:
+      return undefined
+  }
+}
+
```
Comment (SpaghettiOverload) [inline src/ledgers/utils.ts:469]: switchLedgerType() again appears here as a locally defined function, but it does different job compared to the ones in App.tsx and ConnectWallet.tsx, please review namings.

File: src/components/Layout/UserInfo.tsx (hunk 1 above)
Comment (stefanpetkov90) [reply src/components/Layout/UserInfo.tsx]: Done

File: src/App.tsx (hunk 2 above)
Comment (stefanpetkov90) [reply src/App.tsx]: Done

File: src/ledgers/utils.ts (hunk 3 above)
Comment (stefanpetkov90) [reply src/ledgers/utils.ts:469]: Done

**Review by SpaghettiOverload — APPROVED** [review]

**Pull request merged**